app/controllers/v0/messages_controller.rb

```diff
@@ -70,11 +70,11 @@ def reply
       message_params[:id] = message_params.delete(:draft_id) if message_params[:draft_id].present?
       create_message_params = { message: message_params }.merge(upload_params)
 
-      if message.uploads.present?
-        client_response = client.post_create_message_reply_with_attachment(params[:id], create_message_params)
-      else
-        client_response = client.post_create_message_reply(params[:id], message_params)
-      end
+      client_response = if message.uploads.present?
+                          client.post_create_message_reply_with_attachment(params[:id], create_message_params)
+                        else
+                          client.post_create_message_reply(params[:id], message_params)
+                        end
```

Comment:
The thing that bugs me the most is the crazy indentation, which is also required because of another rubocop rule. The actual assignment of the variable is less of a problem, but I'd argue that assigning the variable outside of the `if` like this makes it jarring to read because we're used to the if beginning on the left side of the line. Ultimately, I just can't think of a good reason for rubocop to enforce this style.